src/lib/list/_list-theme.scss

```diff
@@ -46,7 +46,8 @@
     }
 
     .mat-subheader {
-      @include mat-typography-level-to-styles($config, body-2);
+      font: mat-font-weight($config, body-2) mat-font-size($config, body-2)
```

Comment:
The `font` declaration is a little fiddly, especially when omitting values. I'd rather have this split into `font-weight`, `font-size` and `font-family`.